src/common/SPELLS/bfa/dungeons/items.js

```diff
@@ -54,6 +54,11 @@ export default {
     name: 'Rezan\'s Gleaming Eye',
     icon: 'ability_hunter_aspectoftheviper',
   },
+  WEBWEAVERs_SOUL_GEM_DAMAGE: { //Webweaver's Soul Gem
```

Comment:
For some reason, the lowercase `s` got here instead 😅 Can you fix this again please? Together with the usage of this in the module itself

Reply:
Okay, the latest commit now features all capital letters for constants.